timemachine/fe/single_topology.py

```diff
@@ -910,12 +911,69 @@ def interpolate_w_coord(w0: float | jax.Array, w1: float | jax.Array, lamb: floa
     )
 
 
+batch_interpolate_harmonic_bond_params = jax.jit(
```

Comment:
debating about putting these functions and the AlignedPotential classes into interpolate.py

Reply:
I think it'd help w/organization considering the size of single_topology

Reply:
I need to think about how to organize this a little better - deferring it